tooling/eslint/README.md

```diff
@@ -24,7 +24,14 @@ npm install --save-dev eslint babel-eslint eslint-config-airbnb eslint-plugin-pr
 
 ## 2. Setup Config File(s)
 
-Instead of running `eslint --init`, I recommend copying the [sample `.eslintrc` file](#sample-eslintrc) into your `$HOME` folder.
+Instead of running `eslint --init`, I recommend copy and pasting the sample `.eslintrc` file manually. The instructions are below: 
+
+* `cd ~`
```

Comment:
Can you change this to 
```
* `cd <your-project-folder>`
```

Reply:
changed! https://github.com/justsml/guides/pull/3/commits/5db1206afe9d17352756e1dd9f8ae04b099c82d5